overlay/default.nix

```diff
@@ -49,7 +49,7 @@ in
     $sctl set-property --runtime user.slice AllowedCPUs="$sysrange"
 
     # run all arguments as a command using systemd-run inheriting path and limited to $shieldrange cpus
-    $srun --nice=-20 --slice shield -EPATH=$PATH --property=AllowedCPUs="$shieldrange" -t -- "$@"
+    $srun --nice=-20 --slice shield -EPATH=$PATH --property=AllowedCPUs="$shieldrange" -t -d -- "$@"
```

Comment:
Does it make sense to also use `--collect` to avoid collecting failed units in our alerts? :sweat_smile:

Reply:
I was thinking about that. on the one hand it would make sense, on the other hand that would (afaiu) delete all logs for the command execution, which could be very helpful for longer-running commands. 

then again those could just be piped into a file. nevermind :joy: